server/pulp/server/db/model/repository.py

```diff
@@ -266,11 +267,15 @@ def skipped_result(cls, repo_id, distributor_id, distributor_type_id, started, c
 
         @param result_code: one of the RESULT_* constants in this class
         @type  result_code: str
+
+        @param message: reason for skipping
```

Comment:
This needs to indicate that it has already been internationalized. Maybe putting that explicitly here in the docstring and changing `message` to be `translated_message`.